Modules/posixcapmodule.c

```diff
@@ -0,0 +1,645 @@
+/* POSIXCAP module implementation */
+
+#define PY_SSIZE_T_CLEAN
+
+#include "Python.h"
+#ifdef MS_WINDOWS
```

Comment:
AFAICT, this module has nothing to do with Windows, in which case there's no reason to include windows.h or winreparse.h.

Reply:
Noted. This was a result of a copy paste I think. I am revising this patch. Like I said it is still draft. Reviewing it now will waste anyone's time.